### ISSUE TITLE
Remote media file not available: http://www.christoph-lauer.de up for sale?

### ISSUE THREAD

**Issue by cxbrooks** [celestialclauer/Sonogram-Visible-Speech#4]
Under MacOS, I downloaded https://github.com/celestialclauer/Sonogram-Visible-Speech/releases/download/v5.0/SonogramMacOs.zip

The splash screen suggests File -> Open remote media file and the URL

http://www.christoph-lauer.de/demo6.wav

is present as a default.  However, that domain seems to be up for sale?

**Comment by celestialclauer** [comment]
Yes, i put it on GitHub…. I have to change this bug with minor priority… The webpage hohe a friend of mine, i have to less time. Did the bug occur while startup?

--
Mit besten Grüßen / With best regards,
Christoph Lauer

mail: 	christophlauer@me.com <mailto:christophlauer@me.com>
mobile: 	0159/01007198
phone:	06871/9221094

**Comment by cxbrooks** [comment]
No worries, it is a small thing, the app looks quite good!  Making this bug have minor priority is fine.

I'm not sure I understand your question:

> Did the bug occur while startup?

The first time I clicked on Sonogram.app, it seemed like nothing happened.  So tried running from the command line with:
`java -jar Sonogram.app/Contents/Resources/Java/Sonogram.jar` 
and the app started up fine.  I then exited the app and when I double clicked on Sonogram.app, the app started up fine.  So, if there is a bug here, I can't reproduce it.

During startup, the welcome screen suggests File -> open remote media file, which is why I tried it.

Here's a screenshot of what I get with File-> open remote media file:
<img width="562" alt="Screen Shot 2020-09-17 at 6 59 34 PM" src="https://user-images.githubusercontent.com/861991/93546608-fe0c3b80-f917-11ea-96be-e9110728376d.png">
Here's the error message:
<img width="685" alt="Screen Shot 2020-09-17 at 6 51 47 PM" src="https://user-images.githubusercontent.com/861991/93546617-03698600-f918-11ea-9c00-6df48a06d18e.png">

**Comment by celestialclauer** [comment]
Yes, thanks, thats ok. That are the files i used for the first versions. But i have to check if i can put them on github.

--
Mit besten Grüßen / With best regards,
Christoph Lauer

mail: 	christophlauer@me.com <mailto:christophlauer@me.com>
mobile: 	0159/01007198
phone:	06871/9221094
